src/Features/CSharp/Portable/CodeRefactorings/UseExplicitOrImplicitType/AbstractUseTypeCodeRefactoringProvider.cs

```diff
@@ -106,11 +106,10 @@ private static async Task<SyntaxNode> GetDeclarationAsync(CodeRefactoringContext
 
         var syntaxFacts = context.Document.GetLanguageService<ISyntaxFactsService>();
 
-        var typeNode = await context.TryGetRelevantNodeAsync<TypeSyntax>().ConfigureAwait(false);
```

Comment:
we already did thsi above (line 96), and this isn't cheap.

Reply:
nit move this up to line 97 and then other uses of type?.Parent can use this local

Reply:
done.